examples/text_classification/Transformers_README.md

```diff
@@ -0,0 +1,46 @@
+# Serving Transformers Bert using TorchServe
+
+#### Prerequisite:
+
+```
+pip install transformers
+pip install torch torchtext 
+pip install torchserve torch-model-archiver
+```
+
+```
+git clone https://github.com/HamidShojanazeri/serve.git
```

Comment:
@HamidShojanazeri Please remove your personal repo name and make the instructions related to TorchServe main repo